src/main/java/com/salesforce/storm/spout/dynamic/persistence/InMemoryPersistenceAdapter.java

```diff
@@ -35,27 +35,23 @@
 public class InMemoryPersistenceAdapter implements PersistenceAdapter {
 
     /**
-     * Used within tests to more easily validate assertions.
+     * In memory store for this adapter's data.
      */
-    public static PersistConsumerStateCallback<String, Integer, Long> persistConsumerStateCallback = (consumerId, partitionId, offset) -> {
-        // No-op by default.
-    };
+    public static Map<String, Long> storedConsumerState = Maps.newHashMap();
```

Comment:
This feels like it'll break as soon as you fire up more than one vspout will it not?  Being static and shared across all instances?

Reply:
Probably just need to make this a concurrent map in both adapters to be safe.